src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -41,15 +57,24 @@ export const ChannelClaimTableRow = ({ claimedChannel, channelType }: ChannelCla
     staleTime: Infinity,
   });
 
+  const channelName = isPublisherChannel ? publisherQuery.data?.name : serialPublicationQuery.data?.name;
+  const pendingChannelQuery = isPublisherChannel ? publisherQuery.isPending : serialPublicationQuery.isPending;
+
   const organizationQuery = useFetchOrganization(claimedChannel.claimedBy.organizationId);
   const organizationName = getLanguageString(organizationQuery.data?.labels);
 
   const publishingPolicy = claimedChannel.channelClaim.constraint.publishingPolicy;
   const editingPolicy = claimedChannel.channelClaim.constraint.editingPolicy;
 
-  const channelName = isPublisherChannel ? publisherQuery.data?.name : serialPublicationQuery.data?.name;
-
-  const pendingChannelQuery = isPublisherChannel ? publisherQuery.isPending : serialPublicationQuery.isPending;
+  const deleteChannelClaimMutation = useMutation({
+    mutationFn: async () => await deleteChannelClaim(customerIdentifier, channelIdentifier),
```

Comment:
Selv om vi må lage noe av id selv, kan vi vel fortsatt begrense litt her da, feks ved å sende customer id inn hit. Da slipper vi hvertfall å lage den på nytt, men må fortsatt legge til `/channel-claim/${channelIdentifier}`
```suggestion
    mutationFn: async () => await deleteChannelClaim(user.customerId, channelIdentifier),
```